Makefile

```
@@ -9,3 +9,6 @@ npm_install:
 
 bundle_install:
 	bundle install
+
+start:
+	nanoc view
```

Good idea. I think `bundle exec`

```suggestion
start:
	bundle exec nanoc view
```